test/e2e/azure/azure_suite_test.go

```
@@ -0,0 +1,149 @@
+
+func TestAzureE2E(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Secrets Store CSI Driver Operator Azure WIF E2E Suite")
+}
+
+var _ = BeforeSuite(func() {
+	if os.Getenv("RUN_AZURE_E2E") != "true" {
+		Skip("Azure WIF e2e not enabled (set RUN_AZURE_E2E=true)")
+	}
+
+	rand.Seed(time.Now().UnixNano())
```

golangci-lint (2.13.2)

[error] SA1019: math/rand.Seed has been deprecated since Go 1.20 and an alternative has been available since Go 1.0: As of Go 1.20 there is no reason to call Seed with a random value. Programs that call Seed with a known value to get a specific sequence of results should use New(NewSource(seed)) to obtain a local random generator.

(staticcheck)